dist/html/header.html

```diff
@@ -32,27 +32,27 @@
         class="cross-service-header__button cross-service-header__button--one-login js-x-header-toggle">
         <span class="cross-service-header__button-content">One Login</span>
```

Comment:
All looks good apart from the dist directory, is the intention to have the dist directory committed as well?

Reply:
Hi Peter, thanks for giving this a look.

Good question! Yes the intention is to have `dist` committed (for now). 
Dist is the stuff we actually want users to copy/import in the formats that might be relevant for them – for example `dist` contains compiled HTML versions of the service header, while `src` is just dev source files whose structure might change, or whose format might not be useful ( for ex RPs might not necessarily be using Nunjucks as a templating engine). 

If this were formally distributed via a package manager I agree `dist` should never be committed onto the `main` branch. However, our current "distribution" method is asking services to read some guidance and copy some code from this repo into their service 😄

Reply:
Thanks @danacotoran  for the explanation.